src/screens/LoginPage/LoginPage.tsx

```diff
@@ -30,6 +30,7 @@ function LoginPage(): JSX.Element {
   const [modalisOpen, setIsOpen] = React.useState(false);
   const [componentLoader, setComponentLoader] = useState(true);
   const [isInputFocused, setIsInputFocused] = useState(false);
+  const [isInputFocusedCon, setIsInputFocusedCon] = useState(false);
```

Comment:
1. Variables could be named better, isInputFocused is ok but isInputFocusedCon doesn't say what the variable is.
2. Why are we tracking the focused state? we can trigger a function to do something on specific cases sure but the code seems verbose and is using memory for something that could be done more effeciently.